mergify_engine/tasks/engine/commands_runner.py

```diff
@@ -32,14 +32,16 @@
 
 
 def load_action(message):
+    """Load an action from a message.
+
+    :return: A tuple with 2 values: the command name and the action."""
     action_classes = actions.get_commands()
     match = COMMAND_MATCHER.search(message)
     if match and match[1] in action_classes:
         action_class = action_classes[match[1]]
         config = action_class.command_to_config(match[2].strip())
         action = voluptuous.Schema(action_class.get_schema())(config)
-        command = "%s%s" % (match[1], match[2])
-        return command, action
+        return match[1], action
```

Comment:
We must return the args here, there are used in user reported messages

Reply:
Ok, I wasn't sure it was really useful to report the arguments too, but you seem opiniated. :)